.gitignore

```diff
@@ -0,0 +1,7 @@
+src/org/zaproxy/zap/extension/bruteforce/resources/help_*
```

Comment:
Yes, exactly. Every time these files have been generated, that's why I put it here. I will remove now, if that's fixed :)